deploy/stacks/pipeline.py

```diff
@@ -452,6 +452,7 @@ def set_quality_gate_stage(self):
                     commands=[
                         'mkdir -p source_build',
                         'mv backend ./source_build/',
+                        'mv config.json ./source_build/',
```

Comment:
[Clarification] - In this step we are uploading the backend code to an S3 Bucket. This step in necessary for the migration stage that accesses the migration scripts on the bucket from the infra account.Now we also need to access the config.json so we need to add it to the bucket